### PR TITLE
ACS-270 Adding Search services into subchart

### PR DESCRIPTION
**Pull request by sridharvellingiri**

### CODE REVIEW

**Comment by sridharvellingiri** [comment]
> Is it a copy of https://github.com/Alfresco/alfresco-search-deployment ?

Yes Alex

File: helm/alfresco-content-services/charts/alfresco-search/requirements.lock [hunk 1]
```diff
@@ -0,0 +1,6 @@
+dependencies:
```
Comment (mteodori) [inline helm/alfresco-content-services/charts/alfresco-search/requirements.lock:1]: now the requirements are embedded this file can be deleted

File: helm/alfresco-content-services/charts/alfresco-search/requirements.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Alfresco Insight Engine brings Alfresco Insight Zeppelin
```
Comment (mteodori) [inline helm/alfresco-content-services/charts/alfresco-search/requirements.yaml:1]: now the requirements are embedded this file can be deleted

**Comment by sridharvellingiri** [comment]
Latest bamboo job for this branch --> https://bamboo.alfresco.com/bamboo/browse/PLAT-ACSHELM320-ADT-6

File: helm/alfresco-content-services/charts/alfresco-search/requirements.lock (hunk 1 above)
Comment (sridharvellingiri) [reply helm/alfresco-content-services/charts/alfresco-search/requirements.lock]: deleted requirements file

File: helm/alfresco-content-services/charts/alfresco-search/requirements.yaml (hunk 2 above)
Comment (sridharvellingiri) [reply helm/alfresco-content-services/charts/alfresco-search/requirements.yaml]: deleted requirements file

**Pull request merged**